src/Controls/src/Core/ResourceDictionary.cs

```diff
@@ -47,9 +47,20 @@ public void SetAndLoadSource(Uri value, string resourcePath, Assembly assembly,
 			//this will return a type if the RD as an x:Class element, and codebehind
 			var type = XamlResourceIdAttribute.GetTypeForPath(assembly, resourcePath);
 			if (type != null)
+			{
 				_mergedInstance = s_instances.GetValue(type, (key) => (ResourceDictionary)Activator.CreateInstance(key));
```

Comment:
Does this produce a warning?
It should, `key` is not annotated here - but `Activator.CreateInstance` requires annotation. Note that we don't have a clean way of annotating this, it will require a suppression one way or another, but first I'd like to confirm this doesn't hit a bug in the analysis somewhere.

Reply:
Yes, this produces a warning and I have a branch that fixes this line and several other warnings. I wanted to focus on the feature flag in this PR and avoid unrelated changes.

Reply:
> it will require a suppression one way or another

I believe this can be fixed just by changing the lambda to `_ => (ResourceDictionary)Activator.CreateInstance(type)`.